lua/nvim-treesitter/configs.lua

```diff
@@ -150,7 +156,7 @@ local function disable_mod_conf_autocmd(mod)
   end
   -- TODO(kyazdani): detach the correct autocmd... doesn't work when using %s, cmd.
```

Comment:
@kyazdani42 does that comment still apply?

Reply:
don't think so, it's quite old